diesel/src/lib.rs

```diff
@@ -69,7 +69,7 @@
 //! a trait called [`Queryable`].
 //!
 //! Diesel maps "Rust types" (e.g. `i32`) to and from "SQL types"
-//! (e.g. [`diesel::types::Integer`]).
+//! (e.g. [`diesel::sql_types::Integer`]).
 //! You can find all the types supported by Diesel in [the `types` module](types).
```

Comment:
This line references the `types` module which is now "empty". I think this should better point to `sql_types` now